examples/analysis_options.yaml

```diff
@@ -1,4 +1,4 @@
-include: package:pedantic/analysis_options.yaml
+include: package:pedantic/analysis_options.1.8.0.yaml
```

Comment:
Is this temporary?

Reply:
Yes, it is. Just want to get things working so the site will start building again. I'll file a bug to update the analysis options.